src/cbson-oid.c

```diff
@@ -18,8 +19,6 @@ int cbson_oid_create(lua_State* L, const char* oid)
   }
 
   cbson_oid_t* ud = lua_newuserdata(L, sizeof(cbson_oid_t));
-
-  ud->oid = malloc(strlen(oid)+1);
   strcpy(ud->oid, oid);
```

Comment:
My bad, this should be changed to strncpy and appending '\0' manually.

Reply:
I noticed it before, strncpy is unnecessary in my opinion. Because this function has checked the length of oid string at the beginning.

Reply:
Oh, right